myapp/handlers/profile-handler.go

```diff
@@ -198,6 +198,48 @@ func (h *Handlers) ProfileByID(w http.ResponseWriter, r *http.Request) {
 		}
 
 		thisUser := h.App.Session.GetInt(r.Context(), "userID")
+		likedSongs, err := h.Models.LikedSongs.GetAllByOneUser(profile.UserID)
+		if err != nil {
+			fmt.Println("Error getting liked songs from user:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		song1, err := h.Models.Songs.Get(likedSongs[0].SongID)
+		if err != nil {
+			fmt.Println("Error getting song:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		song2, err := h.Models.Songs.Get(likedSongs[1].SongID)
+		if err != nil {
+			fmt.Println("Error getting song:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		song3, err := h.Models.Songs.Get(likedSongs[2].SongID)
+		if err != nil {
+			fmt.Println("Error getting song:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		song4, err := h.Models.Songs.Get(likedSongs[3].SongID)
+		if err != nil {
+			fmt.Println("Error getting song:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		song5, err := h.Models.Songs.Get(likedSongs[4].SongID)
+		if err != nil {
+			fmt.Println("Error getting song:", err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		fullName1 := song1.Name + "  by  " + song1.ArtistName
+		fullName2 := song2.Name + "  by  " + song2.ArtistName
+		fullName3 := song3.Name + "  by  " + song3.ArtistName
+		fullName4 := song4.Name + "  by  " + song4.ArtistName
+		fullName5 := song5.Name + "  by  " + song5.ArtistName
```

Comment:
This causes a panic on my machine, not enough artists provided, page is unable to load at all with my account.

We need a default value provided or should be iterating over an array.